src/plugins/links/public/_mixins.scss

```diff
@@ -25,7 +25,7 @@
 }
 
 @mixin euiFlyout {
-  @include kibanaFullBodyHeight();
+  height: calc(100vh - var(--euiFixedHeadersOffset, 0));
```

Comment:
This fixes a styling issue in Serverless where the height of the secondary edit/add link flyout was incorrect:

| Before | After |
|--------|--------|
| ![image](https://github.com/elastic/kibana/assets/8698078/0d32bda7-a5bc-4655-80ac-bffab6fce61a) | ![image](https://github.com/elastic/kibana/assets/8698078/78f32642-550b-475f-bbf1-12a38189592e) |